TestON/tests/MultiProd/MultiProd.py

```diff
@@ -53,18 +53,19 @@ def CASE1( self, main ):
         removeLogResult = main.ONOSbench.onosRemoveRaftLogs()
 
         main.step( "Git checkout, pull and get version" )
-        #main.ONOSbench.gitCheckout( "master" )
+        main.ONOSbench.gitCheckout( "master" )
         gitPullResult = main.ONOSbench.gitPull()
         main.log.info( "git_pull_result = " + str( gitPullResult ))
         versionResult = main.ONOSbench.getVersion( report=True )
 
-        if gitPullResult == 1:
+        packageResult = main.TRUE
+        if gitPullResult == 100:
```

Comment:
I would suggest making git pull configurable from the params file. We shouldn't assume these tests will always be run in a certain order.